lib/prepare_release.js

```diff
@@ -386,9 +386,10 @@ export default class ReleasePreparation {
                           ` ${releaseInfo}, @${username}\n`;
 
     const topHeader =
-      `<a href="#${lastRef.substring(1)}">${lastRef.substring(1)}</a><br/>`;
+      `<b><a href="#${lastRef.substring(1)}">${lastRef.substring(1)}</a></b><br/>`;
```

Comment:
We may need to make this optional for it to work with all release lines.
Compare
- https://github.com/nodejs/node/blob/main/doc/changelogs/CHANGELOG_V16.md?plain=1
- https://github.com/nodejs/node/blob/main/doc/changelogs/CHANGELOG_V18.md?plain=1
- https://github.com/nodejs/node/blob/main/doc/changelogs/CHANGELOG_V20.md?plain=1

Reply:
what if we just drop the <b> on v20?

Reply:
> what if we just drop the `<b>` on v20?

SGTM as well.